logic/src/commonMain/kotlin/com/wire/kalium/logic/data/connection/ConnectionRepository.kt

```diff
@@ -217,7 +217,8 @@ internal class ConnectionDataSource(
                         selfUserTeamId = selfUser.teamId?.value,
                         otherUserTeamId = userProfileDTO.teamId,
                         selfUserDomain = selfUser.id.domain,
-                        permissionCode = null
+                        permissionCode = null,
+                        userProfileDTO.service != null
```

Comment:
What do you think of using named parameters for this creation logic? I doubted for a second why we were doing a conditional check 😅 , but later realized that is a boolean argument.

Reply:
I don't have idea how I can make it more readable, because list of arguments gets bigger 😅